scripts/UnioneApiChecker.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace UnioneScripts;
+
+use Composer\Script\Event;
+use Unione\UnioneClient;
+
+class UnioneApiChecker
+{
+    /**
+     * The UnioneClient client.
+     *
+     * @var UnioneClient
+     */
+    private UnioneClient $client;
+
+    /**
+     * The script config file.
+     *
+     * @var array
+     */
+    private $config;
+
+    /**
+     * The array with error messages.
+     *
+     * @var array
+     */
+    private $messages = [];
+
+    public function __construct(array $args, array $config)
+    {
+        $this->client = new UnioneClient($args[1], $args[0]);
+        $this->config = $config;
+    }
+
+        /**
+         * Check Unione API methods.
+         *
+         * @param \Composer\Script\Event $event
+         *
+         * @return void
+         */
+        public static function testApi(Event $event)
+        {
+            $args = $event->getArguments();
+
+            if (empty($args) || \count($args) < 2) {
```

Comment:
The first condition is not necessary since getArguments() always returns an array.